framework/db/sqlite/Schema.php

```diff
@@ -16,7 +17,6 @@
 use yii\db\Expression;
 use yii\db\ForeignKeyConstraint;
 use yii\db\IndexConstraint;
-use yii\db\SqlToken;
```

Comment:
This looks incorrect. `SqlToken` is used in typehint:
https://github.com/yiisoft/yii2/blob/2bdfac4ab7af3f68df494eab450ad4cd971ff7e5/framework/db/sqlite/Schema.php

Reply:
https://github.com/yiisoft/yii2/commit/51fc314f585ee28dd9ce5a4fe5ed5ba9eefabb89